src/app/dataset_storage.py

```diff
@@ -1,9 +1,18 @@
 import json
+import os
 
 from google.cloud import storage
 
 storage_client = storage.Client()
 
+# If the local development is used then, it will use those values to create the local storage.
```

Comment:
I don't think you want to do this here because the whole point is to create the bucket object which gets overwritten by the get_dataset function anyway. You could move this whole code block to somewhere else, possible inside your devtools directory.

Reply:
Please remove this section as it causes the build to break.